cli/src/commands/deploy.ts

```diff
@@ -115,8 +123,15 @@ export default class Deploy extends Command {
     return app.id;
   }
 
+  public log(message?: string, ...args: any[]) {
```

Comment:
(not necessarily this PR): I wonder if it makes sense to move this to the `BaseCommand` class`.